core/jobs.py

```diff
@@ -642,7 +649,11 @@ def run(self, job_id, job_class_str, output):
 
 class GoogleCloudStorageConsistentJsonOutputWriter(
         output_writers.GoogleCloudStorageConsistentOutputWriter):
-
+    """This is an Output Writer which is used to consistently store MapReduce
+    job's results in json format. GoogleCloudStorageConsistentOutputWriter is
+    preferred as it's consistent. For more details please look here
+    https://github.com/GoogleCloudPlatform/appengine-mapreduce/wiki/3.4-Readers-and-Writers#googlecloudstorageoutputwriter
```

Comment:
Just checking, this doesn't trigger a linter "line-too-long" issue?

Reply:
no, i ran pre_commit_linter.py and it's fine.

Reply:
@seanlip and @GanitGenius: This line has 118 characters. It is not detected by Pylint since it is one single link. If we break the link by adding a whitespace in between, the linter throws up the "line-too-long" error (I tested it locally here as well as for some other docstrings). Thanks!